snapshot/bundlestore/server.go

```diff
@@ -58,7 +67,28 @@ func (s *Server) HandleUpload(w http.ResponseWriter, req *http.Request) {
 		fmt.Fprintf(w, "Bundle %s already exists\n", bundleName)
 	}
 
-	if err := s.store.Write(bundleName, bundleData); err != nil {
+	// Get ttl if defaults were provided during Server construction or if it comes in this request header.
+	var ttl *TTLValue
```

Comment:
is ttl in the expiration date or span to keep alive?  If its expiration date we should call it that.